src/main/java/org/umssdiplo/automationv01/core/managepage/SHAssignation.java

```diff
@@ -1,4 +1,36 @@
 package org.umssdiplo.automationv01.core.managepage;
 
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
 public class SHAssignation extends BasePage {
-}
+    @FindBy(name = "assignment_button")
+    private WebElement assignmentTab;
+
+    @FindBy(name = "new_assignment")
+    private WebElement addAssignmentButton;
+
+    @FindBy(name = "save")
+    private WebElement acceptNewAssignment;
+
+    @FindBy(className = "swal2-confirm swal2-styled")
```

Comment:
it is not clear, when you read it, if in this page it works, I dont have any observation
but if you will use cssSelector, or xpath , please do it.